src/status_im/ui/screens/communities/requests_to_join.cljs

```diff
@@ -11,19 +11,20 @@
             [status-im.ui.components.icons.icons :as icons]
             [status-im.ui.components.topbar :as topbar]
             [status-im.i18n.i18n :as i18n]
-            [status-im.communities.core :as communities]))
+            [status-im.communities.core :as communities]
+            [quo.components.animated.pressable :as animation]))
 
 (defn hide-sheet-and-dispatch [event]
   (>evt [:bottom-sheet/hide])
   (>evt event))
 
 (defn request-actions [community-id request-id]
   [react/view {:flex-direction :row}
-   [react/touchable-highlight {:on-press #(re-frame/dispatch [:communities.ui/accept-request-to-join-pressed community-id request-id])}
+   [animation/pressable {:on-press #(re-frame/dispatch [:communities.ui/accept-request-to-join-pressed community-id request-id])}
```

Comment:
When `react/touchable-highlight` is used for button _(child)_, then its on-press is also calling `on-press` of `quo/list-item` _(parent)_
Seems, imports from `react` or `react-native` library doesn't work well with `react-native-gesture-handler` library

https://stackoverflow.com/a/59570332